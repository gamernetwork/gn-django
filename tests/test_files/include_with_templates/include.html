<h1><code>include_with</code> test</h1>

<h2>Parent:</h2>
<p>Parent context only: {{ parent_context }}</p>
<p>Out of context: {{ out_of_context }}</p>
{%
  include_with 'i1.html'
  foo=obj,
  hard='this is a hard coded message',
  var=string
%}
{% include_with 'i2.html' foo={'sheep': 'baa', 'cow': 'moo'}, baes=('Link', 'Zelda', 'Seigmeyer'), games=['Dark Souls', 'Bloodborne'], leet=1337 %}
