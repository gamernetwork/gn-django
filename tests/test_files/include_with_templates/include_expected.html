<h1><code>include_with</code> test</h1>

<h2>Parent:</h2>
<p>Parent context only: Parent context</p>
<p>Out of context: {{ out_of_context }}</p>
<h2>First include:</h2>
<p>Object:</p>
<ul>
  <li>Name: Hello World!</li>
  <li>Text: This is text that belongs to a model</li>
</ul>
<p>Hard coded string: this is a hard coded message</p>
<p>Variable string: This is a message</p>
<h2>Second include:</h2>
<p>Dictionary:</p>
<ul>
  <li>sheep: baa</li>
  <li>cow: moo</li>
</ul>
<p>Tuple:</p>
<ul><li>Link</li><li>Zelda</li><li>Seigmeyer</li></ul>
<p>List:</p>
<ul><li>Dark Souls</li><li>Bloodborne</li></ul>
<p>Int: 1337</p>
<p>Parent context only: {{ parent_context }}</p>
<h2>Nested include</h2>
<p>A variable: I&#39;m a barbie girl, in a barbie world</p>
